src/test/java/uk/gov/justice/probation/courtcasematcher/pact/HearingRestClientPactTest.java

```diff
@@ -33,7 +33,7 @@
 @ExtendWith({PactConsumerTestExt.class})
 @PactTestFor(providerName = "court-case-service", port = "8090")
 @PactDirectory(value = "build/pacts")
-class CourtCaseRestClientPactTest {
+class HearingRestClientPactTest {
```

Comment:
This is testing `CourtCaseRestClient` which refers to the `court-case-service` so I think this can be left as CourtCaseRestClientPactTest.

Reply:
There are other areas in this file where court case should perhaps be changed to hearing but fixing it would involve changes to both repos so I'd say leave this one alone other than changes internal to the matcher.